packages/circuit-ui/components/AutocompleteInput/AutocompleteInput.stories.tsx

```diff
@@ -316,6 +325,44 @@ Grouped.args = {
 };
 Grouped.play = openAutocomplete();
 
+export const WithMultiSelection = (args: AutocompleteInputProps) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [autocompleteValue, setAutocompleteValue] = useState<
+    AutocompleteInputOption[]
+  >((args.value ?? []) as AutocompleteInputOption[]);
+  const [options, setOptions] = useState(args.options);
+  const onSearchTextChange = (searchText: string) => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setOptions(filterOptions(searchText, args.options));
+      setIsLoading(false);
+    }, 1500);
+  };
+  const onSelection = (value?: AutocompleteInputOption) => {
+    setAutocompleteValue(
+      updateMultipleSelectionValue(autocompleteValue, value),
+    );
+  };
```

Comment:
The example should demonstrate resetting the options after selection as is recommended in the docs.

```suggestion
    setOptions(args.options);
  };
```